src/FMBot.Bot/Builders/FriendBuilders.cs

```diff
@@ -388,4 +389,56 @@ public async Task<ResponseModel> RemoveFriendsAsync(ContextModel context, string
 
         return response;
     }
+
+    public async Task<ResponseModel> FriendedAsync(ContextModel context)
+    {
+        var response = new ResponseModel
+        {
+            ResponseType = ResponseType.Embed,
+        };
+
+        if (context.DiscordGuild != null)
+        {
+            response.Embed.WithDescription("This command is only supported in DM");
+            response.CommandResponse = CommandResponse.Error; // unsure if some other type should be used
+            return response;
+        }
+
+        var friended = await this._friendsService.GetFriendedAsync(context.ContextUser.UserNameLastFM);
+
+        if (friended?.Any() != true)
+        {
+            response.Embed.WithDescription("It doesn't seem like anyone's added you as a friend yet.");
+            response.CommandResponse = CommandResponse.NotFound;
+            return response;
+        }
+
+        response.EmbedAuthor.WithName("People who have added you as a friend in .fmbot");
+
+        var pages = new List<PageBuilder>();
+
+        var friendedPages = friended.ChunkBy(10);
+        var counter = 1;
+        var pageCounter = 1;
+        foreach (var friendedPage in friendedPages)
+        {
+            var friendedPageString = new StringBuilder();
+
+            foreach (var friend in friendedPage)
+            {
+                friendedPageString.AppendLine($"{counter}. **{friend.User.UserNameLastFM}**");
```

Comment:
You can make this a profile link with one of the extensions provided in `LastFmUrlExtensions`